### PR TITLE
fix(complete): allows description about command with colon

### PR DESCRIPTION
**Pull request by imcuttle**
See https://github.com/mklabs/tabtab/issues/43 for more information

### CODE REVIEW

**Comment by mklabs** [comment]
Looks really cool ;) As stated in #43 I need to test it before merging but lacks the time to do so right now. I'll try to keep you updated but I really appreciate your PR. Thanks a lot!

File: lib/complete.js [hunk 1]
```diff
@@ -228,10 +228,13 @@ class Complete extends EventEmitter {
...
-    var name = parts[0];
-    var desc = parts.slice(-1)[0];
-
+    var name = str
+    var desc = ''
+    var matching = /^(.*?)(\\)?:(.*)$/.exec(str)
+    if (matching) {
+      name = matching[1]
+      desc = matching[3]
+    }
```
Comment (mklabs) [inline lib/complete.js]: Please add a semi colon, I'm still using them in this project ;)

Don't worry about that, I'll try to merge your PR today and will add them.

Comment (imcuttle) [reply lib/complete.js]: Finished

Comment (mklabs) [reply lib/complete.js]: Awesome, thanks @imcuttle. I'm actually currently working on tabtab, trying to test your PR on bash for now.

**Comment by mklabs** [comment]
Thank you for your PR @imcuttle

**Pull request merged**